### PR TITLE
OSDOCS-11741: trouble shooting cleanup  data added to release notes

### PR DESCRIPTION
**Pull request by aravipra**
Version(s):
4.17

Issue:
[OSDOCS-11741](https://issues.redhat.com/browse/OSDOCS-11741)

Link to docs preview:
[Data cleanup now documented](https://80750--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-17-release-notes.html#microshift-4-17-doc-enhancements_release-notes)

QE review:
- [x] QE has approved this change.

SME review:
- [x] SME has approved this change.

### CODE REVIEW

File: microshift_release_notes/microshift-4-17-release-notes.adoc [hunk 1]
```diff
@@ -97,6 +97,11 @@ With this release, the resource footprint of the LVM storage driver is significa
 === Documentation enhancements
 //Doc enhancements include additions for RFEs and other continuous improvement items that are substantial and are not tied to new features or bug fixes already listed here. These can also go under the relevant section as applicable.
 
+[id="microshift-4-17-data-cleanup_{context}"]
+==== Data cleanup now documented
+With this release, various troubleshooting tasks to cleanup data are documented. For more information, see
```
Comment (ggiguash) [inline microshift_release_notes/microshift-4-17-release-notes.adoc:102]:
```suggestion
With this release, the `microshift-cleanup-data` script is documented. For more information, see
```

Comment (aravipra) [reply microshift_release_notes/microshift-4-17-release-notes.adoc]: done

**Comment by ggiguash** [comment]
LGTM from Engineering

**Review by kcarmichael08 — COMMENTED** [review]
LGTM!

**Review by jldohmann — APPROVED** [review]
nice work! LGTM

**Pull request merged**